src/exm-window.c

```diff
@@ -243,18 +251,26 @@ extension_install (GtkWidget  *widget,
     {
         GtkWidget *dlg;
 
-        dlg = gtk_message_dialog_new (GTK_WINDOW (self),
-                                      GTK_DIALOG_MODAL,
-                                      GTK_MESSAGE_QUESTION,
-                                      GTK_BUTTONS_YES_NO,
-                                      _("This extension does not support your GNOME Shell version.\nWould you like to install anyway?"));
+        dlg = adw_message_dialog_new (GTK_WINDOW (self),
+                                      _("Install Anyway?"),
+                                      _("This extension does not support your GNOME Shell version. Are you sure you want to install?"));
+
+        adw_message_dialog_add_responses (ADW_MESSAGE_DIALOG (dlg),
+                                          "no", _("_No"),
+                                          "yes", _("_Yes"),
+                                          NULL);
+
+        adw_message_dialog_set_response_appearance (ADW_MESSAGE_DIALOG (dlg), "no", ADW_RESPONSE_SUGGESTED);
```

Comment:
Hmm, I'm unsure about this one.

It makes sense logically that 'No' would be the suggested option, but it feels strange to have no be highlighted in blue and on the left hand side.

It might be worth inverting the selection and having:

> **Unsupported extension**
> This extension does not support your GNOME Shell version. It may cause errors if installed.
> 
> \> Install Anyway (destructive)
> \> Go Back (suggested)

It would match the HTTP security warning that some browsers give, plus make it a bit clearer that the user is entering unsupported territory. Thoughts?

Reply:
My other option was to remove the suggested style because it also looked weird to me, but it looks much better that way:

![Captura desde 2023-08-29 10-50-21](https://github.com/mjakeman/extension-manager/assets/42654671/4e1a583c-d87b-4a3b-95c8-e980cb3034e1)

Reply:
> It might be worth inverting the selection

Sorry, I just realized this, it should be ready now and it makes sense to have it that way.

![image](https://github.com/mjakeman/extension-manager/assets/42654671/24200cf4-4681-4318-a63e-5fa1eadf12b8)